### PR TITLE
Added right-click directive and attribute

### PR DESCRIPTION
**Pull request by Chris0lsen**
I had need of right-click capabilities, so I added a directive locally to MDP to handle this case.

How to use:

- Declare a function in your controller and assign it to a scope variable. The function should accept the arguments (event,day).
- Assign this variable to the "right-click" attribute of your multiple-date-picker. Don't include the parentheses in the attribute!

e.g. `<multiple-date-picker right-click="myFunction"></multiple-date-picker>`

Working plunk: http://plnkr.co/edit/NlPtbdMhFnfe7OMm2d9N?p=preview

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ gulp.task('lint-app', function () {
         .pipe(jshint.reporter('fail'));
 });
 
-gulp.task('uglify', ['lint-app'], function () {
```
Comment (mgohin) [inline gulpfile.js]: Why did you remove lint-app ?

Comment (Chris0lsen) [inline gulpfile.js]: I didn't look carefully at the gulpfile, and didn't see that the task was intended to fail if linting errors were encountered :) I'll add it back in, it was an oversight on my part.

**Comment by mgohin** [comment]
Merged it with some refacts, thanks :)

**Comment by Chris0lsen** [comment]
Thanks, glad I could help :)
